packages/manager/core/generator/app/conditional-templates/onboarding/index.tsx.hbs

```diff
@@ -19,22 +27,30 @@ export default function Onboarding() {
     ],
   })[0] as UseQueryResult<any>;
   {{/if}}
+  const environment = useEnvironment();
+  const region = environment.getRegion();
+  const guide = constants[region];
+  const { subsidiary } = useAuthentication();
```

Comment:
**nitpick:** Guides are maintained per subsidiary
```
FR: '....',
IT: '....'
AU: '....'
```
So, implementation must be changed to something as below 
```suggestion
  const { subsidiary } = useAuthentication();
  const guide = constants[subsidiary];
```